docs/api/version-history.md

```diff
@@ -18,6 +18,8 @@ keywords: "API, Docker, rcli, REST, documentation"
 [Docker Engine API v1.36](https://docs.docker.com/engine/api/v1.36/) documentation
 
 * `Get /events` now return `exec_die` event when an exec process terminates.  
+* `POST /containers/create` now takes `HostConfig.Init` field to run an init
```

Comment:
I think this option was actually added in an older API version already (but don't know which one it was) could you check that?